### ISSUE TITLE
Division by zero in Galois fields

### ISSUE THREAD

**Issue by dimitrim2010** [Macaulay2/M2#1488]
M2 divides by zero without notice

```m2
1/0
p=167
K=GF(p^2,Variable=>a)
1_K/0_K
K0=GF(ZZ/p[a]/(a^2-a+2), Variable=>a)
1_K0/0_K0
--
-- Works better with the same field produced by "toField"
--
K1=toField (ZZ/p[a]/(a^2-a+2));
1/a
a/0
1_K1/0_K1

i1 : 1/0
stdio:1:2:(3): error: division by zero

i2 : p=167

o2 = 167

i3 : K=GF(p^2,Variable=>a)

o3 = K

o3 : GaloisField

i4 : 1_K/0_K

o4 = 0

o4 : K

i5 : K0=GF(ZZ/p[a]/(a^2-a+2), Variable=>a)

o5 = K0

o5 : GaloisField

i6 : 1_K0/0_K0

o6 = 0

o6 : K0

i7 : --
     -- Works better with the same field produced by "toField"
     --
     K1=toField (ZZ/p[a]/(a^2-a+2));

i8 : 1/a

o8 = 83a - 83

o8 : K1

i9 : a/0
stdio:12:2:(3): error: division by zero

i10 : 1_K1/0_K1
stdio:13:5:(3): error: either element not invertible, or no method available to compute its inverse
```

**Comment by mahrud** [comment]
This is not a comment about the bug itself, which seems bad, but you can use ```` ``` ... ``` ```` to encapsulate the code text in a more readable format. There's also a button that looks like `< >` in the GitHub interface. To also use syntax highlighting, do something like this:
````m2
```m2
-* your code here *-
C = res M
if X then Y else Z
```
````

**Comment by DanGrayson** [comment]
Thanks!

```m2
Macaulay2, version 1.16.0.2
with packages: ConwayPolynomials, Elimination, IntegralClosure, InverseSystems, LLLBases, MinimalPrimes, PrimaryDecomposition, ReesAlgebra, TangentCone, Truncations

i1 : k=GF 3

o1 = k

o1 : GaloisField

i2 : 1/0_k

o2 = 0

o2 : k
```

**Comment by DanGrayson** [comment]
fixed in commit 5af9263701ac75197f0398e822a8c9b621060d4d
